src/rdma.c

```diff
@@ -0,0 +1,1931 @@
+/* ==========================================================================
+ * rdma.c - support RDMA protocol for transport layer.
+ * --------------------------------------------------------------------------
+ * Copyright (C) 2021-2024  zhenwei pi <pizhenwei@bytedance.com>
+ *
+ * This work is licensed under License 1 of the COPYING file in
+ * the top-level directory.
+ * ==========================================================================
+ */
+
+#define VALKEYMODULE_CORE_MODULE
+#include "server.h"
+
+#ifdef USE_RDMA
+#ifdef __linux__    /* currently RDMA is only supported on Linux */
+#include "connection.h"
+#include "connhelpers.h"
+#include "cluster.h"
```

Comment:
Why do you need `cluster.h` here?

Reply:
It is included wrongly. Remove it in the new version.

And use clang-format code in the new version.